src/main/WorkflowSelection.tsx

```diff
@@ -76,7 +76,7 @@ const WorkflowSelection : React.FC = () => {
     return (
       <div css={workflowSelectionStyle}>
         <h2>{topTitle}</h2>
-        {topText}
+        {!!topText}
```

Comment:
Hu, what's that for? You convert it to a boolean, but rendering booleans in JSX results in nothing.

Reply:
I don't even know. Will undo.